pkg/scheduler/scheduler.go

```diff
@@ -252,6 +253,7 @@ func (s *Scheduler) admit(ctx context.Context, e *entry) error {
 		}
 		log.Error(err, "Could not admit workload and assigning flavors in apiserver")
 		s.requeueAndUpdate(log, ctx, &e.Info, err.Error())
+		s.recorder.Eventf(newWorkload, corev1.EventTypeNormal, "Pending", "Could not admit workload and assigning flavors in apiserver")
```

Comment:
add `: %v", err)`

sorry it wasn't clear

Reply:
for a second I thought so, but.... decided otherwise ;P